### DISCUSSION TITLE
How to add a new language to an XTTS v2 model

### DISCUSSION THREAD

**Discussion by bobur-amirov** [coqui-ai/TTS#3689] [General Q&A]
How to add a new language to an XTTS v2 model?

**Comment by StephennFernandes** [comment]
Looking for similar solution. Did you find anything ?

**Comment by sanebg** [comment]
Hey I'm interested too. I have a massive royalty free studio-quality dataset in Bulgarian (50 hours) I also have 2xRTX 4090 and I'm ready to train. If somebody can give me instructions, I'd like to contribute to the Bulgarian language!

**Reply by StephennFernandes** [reply]
50 hrs of studio quality data 😲🤯

That's insanely great, you'll get an amazing TTS model on this given that you well train and refine all the hparam's well.

**Reply by sanebg** [reply]
It's 48 hours to be exact, but yea. I have 48 hours of royalty free audiobooks recorded in my studio. Female and male. However I have no idea how to train a new language in Coqui.

**Reply by hrachkovinovoto** [reply]
> Hey I'm interested too. I have a massive royalty free studio-quality dataset in Bulgarian (50 hours) I also have 2xRTX 4090 and I'm ready to train. If somebody can give me instructions, I'd like to contribute to the Bulgarian language!

Ето нещо ново и се надявам да успееш! Добави ме в discord-а когато си свободен: blastershot92z#3737
https://github.com/nguyenhoanganh2002/XTTSv2-Finetuning-for-New-Languages

**Reply by LifeImprovingInnovations** [reply]
Здравей, Ице!
Ако все още имаш интерес за добавяне на качествена българска реч, то може да се пробваш тук - https://github.com/SWivid/F5-TTS
Инструкции - https://www.youtube.com/watch?v=GmketyZW2c4

**Reply by jcllobet** [reply]
my wife is Bulgarian and would be absolutely awesome to surprise her with this. Any updates on this? I would be happy to collaborate :)

**Comment by taz4et** [comment]
Have you experimented with [alltalk tts](https://github.com/erew123/alltalk_tts)? I found it excellent for fine-tuning, as it streamlines the process considerably. In theory, you could add a new language to xtts-v2 and then fine-tune the model using your own dataset 🤔 

Edit: [Training a Model](https://docs.coqui.ai/en/latest/training_a_model.html)
